py-polars/tests/test_udfs.py

```diff
@@ -65,6 +65,11 @@
     ("a", lambda x: MY_CONSTANT + x, 'MY_CONSTANT + pl.col("a")'),
     ("a", lambda x: 0 + numpy.cbrt(x), '0 + pl.col("a").cbrt()'),
     ("a", lambda x: np.sin(x) + 1, 'pl.col("a").sin() + 1'),
+    (
+        "a",  # note: functions operate on consts
+        lambda x: np.sin(3.14159265358979) + (x - 1) + abs(-3),
+        '(np.sin(3.14159265358979) + (pl.col("a") - 1)) + abs(-3)',
+    ),
```

Comment:
could we add a test case with `abs(x)` to `TEST_CASES` too please?

Reply:
Done 👌